src/models.py

```diff
@@ -421,6 +427,12 @@ def build_task_specific_modules(
     ''' Build task-specific components for a task and add them to model.
         These include decoders, linear layers for linear models.
      '''
+    assert not (args.skip_embs == 1 and args.sep_elmo_embs_for_skip == 0 and args.sent_enc == "null"), \
+        "Currently, skip_embs = 1, sep_elmo_embs_for_skip=1, and sent_enc = null, which means " \
+        "you're feeding in the contextual embeddings twice into your model. Please change settings"
```

Comment:
This seems to send a different message from the assert I commented on above. What does sent_enc = null do?

Reply:
sent_enc = null means f(x) = x. What this assert basically says is that it doesn't make sense to pass in skip_embs and also have sent_enc = null, since that means we're passing into the classifier the output of the contextual encoder twice (once from the sent_enc = null, and once from the skip embedding).